src/calculateRentalCost.js

```diff
@@ -3,8 +3,19 @@
  *
  * @return {number}
  */
+
+const rentCost = 40;
+const rentDiscont3Days = 20;
+const rentDiscontWeek = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if ((days >= 3) & (days < 7)) {
```

Comment:
The condition `(days >= 3) & (days < 7)` uses the bitwise AND operator `&` instead of the logical AND operator `&&`. This is a logical error. You should replace `&` with `&&` to correctly evaluate the condition.